송수경/4월 2주차/BOJ_G5_4811_알약.java

```diff
@@ -0,0 +1,32 @@
+package boj.Gold;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class BOJ_G5_4811_알약 {
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		StringBuilder sb = new StringBuilder();
+		
+		long[][] dp = new long[31][31];
+		for(int i=0; i<31; i++) dp[i][0]=1;
+		int N;
+		
+		for(int i=0; i<31; i++) {
+			for(int j=0; j<31; j++) {
+				if(i<j || j==0) continue;
```

Comment:
`if` 문 조건을 `for` 의 조건으로 넣을 수 있지 않을까요?
```suggestion
			for(int j = 1; j <= i; j++) {
```